useful-scripts/misc-test-utils/src/services/commonService.ts

```diff
@@ -0,0 +1,202 @@
+      );
+    }
+
+    if (fs.existsSync(`${cwd}/test-files`)) {
+      rimraf(`${cwd}/test-files/*`, () => {
+        console.log(chalk.green('cleaned test-files folder'));
+      });
+    }
+
+    if (fs.existsSync(`${cwd}/zip-files`)) {
+      rimraf(`${cwd}/zip-files/*`, () => {
+        console.log(chalk.green('cleaned zip-files folder'));
+      });
+    }
+
+    if (!fs.existsSync(`${cwd}/test-files`)) {
+      fs.mkdirSync(`${cwd}/test-files`);
+    }
+
+    if (!fs.existsSync(`${cwd}/zip-files`)) {
+      fs.mkdirSync(`${cwd}/zip-files`);
+    }
+
+    if (zipGlob[1]) {
+      rimraf(zipGlob[1], () => {
+        console.log(chalk.green('cleaned stale zip files'));
+      });
+    }
+  },
+  createRelease: async (publicationId: string): Promise<string | null> => {
+    console.time('createRelease');
+    try {
+      const res = await axios({
+        method: 'POST',
+        url: `${ADMIN_URL}/api/publications/${publicationId}/releases`,
+        data: {
+          timePeriodCoverage: { value: 'AY' },
+          releaseName: 2222,
+          typeId: '1821abb8-68b0-431b-9770-0bea65d02ff0',
+          publicationId: 'c76c1bc4-0a01-4116-74cd-08d8eebbf0c1',
+          templateReleaseId: '',
+        },
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${JWT_TOKEN}`,
+        },
+      });
+
+      console.timeEnd('createRelease');
+
+      const releaseId = res.data.id;
+
+      console.log(
+        chalk.green(
+          `Release URL: ${ADMIN_URL}/publication/${publicationId}/release/${releaseId}/data`,
+        ),
+      );
+      return releaseId;
+    } catch (e) {
+      errorHandler(e);
+    }
+    return null;
+  },
+
+  extractZip: async () => {
+    const zippedFile = await globby(`${cwd}/*.zip`);
+    const cleanZipFile = zippedFile[0];
+    const archive = fs.existsSync(zippedFile[0]);
+    if (archive) {
+      // eslint-disable-next-line new-cap
+      const zip = new StreamZip.async({ file: cleanZipFile });
+      const count = await zip.extract(null, `${cwd}/test-files`);
+      console.log(chalk.green(`Extracted ${count} entries to test-files`));
+      await zip.close();
+    }
+  },
+  renameFiles: async () => {
+    // define a random id to assign to the renamed data & meta file...
+    const randomId = Math.floor(Math.random() * 1000000);
+
+    // rename csv file..
+    const csvGlob1 = await globby(`${cwd}/test-files/*.csv`);
+
+    const csvGlob2 = csvGlob1[0];
+
+    fs.rename(`${csvGlob2}`, `${cwd}/test-files/testfile-${randomId}.csv`, e =>
+      e ? console.error(chalk.red(e)) : '',
+    );
+
+    // rename the meta file...
+    const metaGlob = await globby(`${cwd}/test-files/*.meta.csv`);
+    const metaGlob2 = metaGlob[0];
+
+    fs.rename(
+      `${metaGlob2}`,
+      `${cwd}/test-files/testfile-${randomId}.meta.csv`,
+      e => (e ? console.error(chalk.red(e)) : ''),
+    );
+  },
+
+  addMetaGuidance: async (
```

Comment:
Would suggest putting this in `releaseService` as meta guidance is a release-specific thing